nc/views.py

```diff
@@ -169,11 +179,58 @@ def contraband_hit_rate(self, request, pk=None):
         q = Q(search_type__isnull=False)
         self.query(results, group_by=("year", "driver_race", "driver_ethnicity"), filter_=q)
         response["searches"] = results.flatten()
-        # searches
+
+        # contraband
         results = GroupedData(by="year", defaults=GROUP_DEFAULTS)
         q = Q(contraband_found=True)
         self.query(results, group_by=("year", "driver_race", "driver_ethnicity"), filter_=q)
         response["contraband"] = results.flatten()
+
+        # # contraband types
+        qs = Contraband.objects.filter(stop__agency=self.get_object())
```

Comment:
This gets tricky, because we only want to report on drivers to match the rest of the site, but currently this will return contraband tied to passengers too.

Reply:
Oh I see, I didn't know passengers were involved as well. Thanks for pointing that out!